### DISCUSSION TITLE
Align terrain detail to terrain

### DISCUSSION THREAD

**Discussion by mroshaw** [visualdesigncafe/nature-renderer#26] [Help] [answered]
Hi! Using Nature Renderer 2022, Unity 2022.3, HDRP 14. My terrain detail isn't aligned to the terrain, it seems to be completely horizontal:
![image](https://github.com/visualdesigncafe/nature-renderer/assets/13573293/51546f21-eb8d-423f-892e-b6554df1c9d8)
Is there a specific setting somewhere that's required to align the detail to the terrain? I can't find anything for that in Global Settings or Object Settings.

Thank you!

**Comment by Swedenkvist** [comment]
I need to know this as well! I was hoping that I could use NR 2022 just like the blanket shader module for The Vegetation Engine but I can't get it to work and there doesn't seem to be any entries on this in the documentation. 

If I use one of the rock Prefabs from the demo, for example, and drag it into the demo scene, it exists there as a game object but isn’t aligned automatically to the terrain. So if I right-click it and select ”Convert to Nature Instance” that makes it a nature instance but it seems to change in size dramatically and becomes much larger. My game requires a workflow where I place objects as game objects procedurally at runtime so I am guessing that I will need to convert them to instances at runtime as well. However, I’m not sure how to do this. And if they change in size when converted, I’m not sure how to Place them properly or even align them to the terrain similar to the Blanket Shaders for TVE.

I have been looking through your documentation but I can’t find anything about aligning to terrain there and there is no option for searching through the docs to find what you need so I've had to look around manually.

**Reply by mroshaw** [reply, marked as answer]
I found it, eventually!
![image](https://github.com/visualdesigncafe/nature-renderer/assets/13573293/05c43811-7ebf-4c9c-9917-f792e1780e96)

**Reply by Swedenkvist** [reply]
Thank you! Didn't seem to work for me, however, since I did convert it and I actually used an asset that was already in the demo so I'm guessing this value was already at max on that prefab... There's very little explanation to these things and it's difficult to get in touch with the developers since they have no Discord, no support email and they don't seem to respond that much on this forum...
